arches/management/commands/packages.py

```diff
@@ -665,8 +665,11 @@ def create_package(self, dest_dir):
 
     @staticmethod
     def update_resource_geojson_geometries():
-        with connection.cursor() as cursor:
-            cursor.execute("SELECT * FROM refresh_geojson_geometries();")
+        if not connection.in_atomic_block:
+            with connection.cursor() as cursor:
+                cursor.execute("SELECT * FROM refresh_geojson_geometries();")
```

Comment:
Can you try removing this workaround in the test framework now?
https://github.com/archesproject/arches/blob/d27ba10b53f8ca05d9c51ed77c9aab1516661b34/tests/base_test.py#L149-L150

That will add some implicit test coverage.